application/src/main/java/org/opentripplanner/graph_builder/module/ned/DegreeGridNEDTileSource.java

```diff
@@ -72,7 +72,8 @@ public List<File> getNEDTiles() {
   }
 
   private String formatLatLon(int x, int y) {
-    String northSouth, eastWest;
```

Comment:
I think this is ok (it is assignment that is problematic) - in some cases where there are a lot of variables a long list take a lot of space. Here is an example: https://github.com/opentripplanner/OpenTripPlanner/blob/1bd0f3833e679e9c26e6a5b88a2848798904bfe0/application/src/test/java/org/opentripplanner/graph_builder/module/transfer/DirectTransferGeneratorTestData.java#L111

Moving the type down to the assignment would be possible, but that reduces the readability - especially since `var` is not a good option here. 

It is only for local variables this is needed - unsure if that is possible to configure.